python-rhsm/src/rhsm/certificate2.py

```diff
@@ -82,7 +82,7 @@ def create_from_pem(self, pem, path=None):
 
     def _read_x509(self, x509, path, pem):
```

Comment:
You are right. The certificate object can be created not only from file (`path`), but it also can be created from string (`pem`).